chain-core/src/property.rs

```diff
@@ -35,6 +35,21 @@
 //! is selected to write a block in the chain.
 //!
 
+use std::hash::Hash;
```

Comment:
Can this be move to the top of the file ?

Reply:
My IDE complained; I understand that module-level doc should go first, and it feels more logical to me too.